com.unity.formats.fbx/Tests/FbxTests/FbxCameraTests.cs

```diff
@@ -102,6 +102,7 @@ public void PhysicalCameraTest()
 
             Assert.That( srcCam.fieldOfView, Is.EqualTo(origCam.fieldOfView).Within(EPSILON));
             Assert.That( srcCam.focalLength, Is.EqualTo(origCam.focalLength));
+            Assert.That( srcCam.aspect, Is.EqualTo(origCam.aspect));
```

Comment:
PhysicalCameraTest attempts to export a physical camera. This calls into `ConfigurePhysicalCamera` in CameraVisitor.cs that executes `GetSizeOfMainView()` to calculate the fbxCamera's aspect ratio.